docs/modules/reference/pages/provisioning/handlers/vmware.adoc

```diff
@@ -184,7 +181,7 @@ Enabling the https://opennms.discourse.group/t/storing-data-with-foreign-sources
 
 == Provision a subset of VMware entities
 
-If you want to provision an arbitrary selection of VMware entities, you can specify a key-value pair to match objects that have a specific attribute in vCenter.
+To provision an arbitrary selection of VMware entities, you can specify a key-value pair to match objects that have a specific attribute in vCenter.
```

Comment:
Are these examples still valid for the v30 importer where there is no XML?

Reply:
Yes, there is an Advanced Options section that lists all of these same parameters. Will need Jeff to verify that the default values are still valid.